posthog/api/capture.py

```diff
@@ -360,10 +374,15 @@ def get_event(request):
         except Exception as e:
             capture_exception(e)
 
+        # TODO we're not going to return 429 on events before we audit our SDKs
```

Comment:
😢 at least the code is there for the future

Reply:
Indeed. Here's my feedback from slack:

- While it will be trivial to check the one recordings SDK for proper behaviour, I’m a bit concerned that we’ll kill some customer backend with an unhandled exception here.
- A first step could be to enable it for posthog-js only, but there’s no easy signal to detect it ($lib is an even prop, not a header, and not sent for snapshots)
- I really appreciate you went generic for this mechanism, but if we don’t filter by lib, shall we start with recordings only for now, and enable the mechanism for events after a quick audit of the official SDKs?
- Also, one of our Q3 goal is to decide and start enforcing some input limits, so we might start rate-limiting on the events/minute/distinct_id, so we’ll definitely investigate 429 handling.